### ISSUE TITLE
card-title stick to the top of the card

### ISSUE THREAD

**Issue by Johann-S** [twbs/bootstrap#18499]
I think it's a bit weird that the card-title don't have a ``margin-top`` 
http://jsfiddle.net/foot0mr2/

But yes I can handle that alone on my own project but I think it would be better with a ``margin-top``

I know my jsfiddle use Bootstrap4 alpha1 but it still present in alpha2

Maybe the solution is to use ``.card-block`` ?

**Comment by twbs-lmvtfy** [comment]
Hi @Johann-S!

You appear to have posted a live example (http://fiddle.jshell.net/foot0mr2/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 52, column 1 thru column 7: End tag for  ```` body ```` seen, but there were unclosed elements.
* line 39, column 3 thru column 25: Unclosed element ```` div ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by mdo** [comment]
> Maybe the solution is to use `.card-block`?

Yup, that or the margin utilities. The title is designed to be within a padded container, but should you need to go another route those two options can help.

Closing as a won't fix for now.
